modules/arm_plugin/thirdparty/CMakeLists.txt

```diff
@@ -151,7 +152,10 @@ else()
         WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/ComputeLibrary
         COMMENT "Build Arm Compute Library"
         DEPENDS ${SOURCES}
-    )
+                ${CMAKE_CURRENT_LIST_FILE}
+                ${ARM_COMPURE_SOURCE_DIR}/SConscript
+                ${ARM_COMPURE_SOURCE_DIR}/SConscript
```

Comment:
Duplicated lines?

Reply:
Fixed, the second line should be `SConstruct`